vortex/core/helpers.py

```diff
@@ -63,3 +66,19 @@ def job_args_match(job, app, args):
         except KeyError:
             matched = False
     return matched
+
+
+def concurrent_job_count_for_tool(app, entity, user=None):  # requires galaxy version >= 21.09
```

Comment:
Since the tool is available in the context, that could perhaps be used instead?
https://github.com/usegalaxy-au/total-perspective-vortex/blob/55bda29bcba72b1dbcbd75fd353860b9021fdb02/vortex/core/mapper.py#L88

Reply:
:) that's much better